ml_model/src/main/java/edu/wm/sealab/featureextraction/FeatureVisitor.java

```diff
@@ -61,6 +63,19 @@ public void visit(ConstructorDeclaration cd, Void arg) {
   public void visit(IfStmt ifStmt, Void arg) {
     super.visit(ifStmt, arg);
     features.incrementNumOfIfStatements();
+    features.incrementNumOfConditionals();;
+  }
+
+  /**
+   * This method to compute # switch statements of a java method (not entries)
+   *
+   * @param SwitchStmt
+   * @param Void
```

Comment:
These `@param` tags are useless if they just name the type (it is already in the method signature). Either add more information here about the parameters or remove them. (This applies elsewhere in this file, as well.)

Reply:
I was following the formatting of the existing methods - @NGimhana :  Do these have any use/reason for existing?

Reply:
I agree with Martin. @Allusionist8359 Every method in the file needs to be fixed as mentioned. Keep in mind that just following the existing without a proper understanding would not always be the best solution because those can be incomplete sometimes.